extensions/2.0/Khronos/KHR_materials_emissiveStrength/README.md

```diff
@@ -0,0 +1,141 @@
+# KHR\_materials\_emissive\_strength
+
+## Contributors
+
+* Ben Houston, Threekit, [@bhouston](https://twitter.com/BenHouston3D)
+* Ed Mackey, Analytical Graphics, [@emackey](https://twitter.com/emackey)
+* Bastian Sdorra, Dassault Systemes [@bsdorra](https://github.com/bsdorra)
+* Alex Wood, Analytical Graphics, [@abwood](https://twitter.com/abwood)
+
+Copyright (C) 2021 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Exclusions
+
+* This extension must not be used on a material that also uses `KHR_materials_pbrSpecularGlossiness`.
+* This extension must not be used on a material that also uses `KHR_materials_unlit`.
+
+## Overview
+
+The core glTF 2.0 material model includes an `emissiveFactor` and an `emissiveTexture` to control the color and
+intensity of the light being emitted by the material, clamped to the range [0.0, 1.0]. However, in PBR environments
+with high-dynamic range reflections and lighting, stronger emission effects may be desirable.
+
+In this extension, a new `emissiveStrength` scalar factor is supplied, that governs the upper limit of emissive
+strength per material. This strength can be colored and tempered using the core `emissiveFactor`
+and `emissiveTexture` controls, permitting the strength to vary across the surface of the material.
+Supplying values above 1.0 for `emissiveStrength` can have an influence on reflections, tonemapping, blooming, and more.
+
+For systems where a physical light unit is needed, the units for `emissiveStrength` are kilonits (thousands of
+nits, where a nit is candela per square meter: **1000 cd/m<sup>2</sup>**).  This is intended to be roughly

```

Comment:
Should `emissiveStrength` be unitless? This parameter is simply a multiplier, and the units are defined on the emissive channel of core glTF.  The result of a multiplication would imply that we are squaring our units.